raiden/main.py

```diff
@@ -1,7 +1,9 @@
 from gevent import monkey
-monkey.patch_all()
+
 from raiden.ui.cli import run
 
+monkey.patch_all()
```

Comment:
The monkey patch call needs to be at the top of the file.

There is an open issue in isort about this: https://github.com/timothycrosley/isort/issues/468

The easiest fix currently seems to be to 'hide' the rest of the imports inside an `if True:` block.